library/Garden/StaticCacheTrait.php

```diff
@@ -2,7 +2,7 @@
 /**
  * @author Alexander Kim <alexander.k@vanillaforums.com>
  * @copyright 2009-2018 Vanilla Forums Inc.
- * @license GPLv2
+ * @license GPL v2
```

Comment:
If we're going to update these we should use the proper one. I know its a bit confusing at the moment because we haven't mass updated them all, but they should be `@license GPL-2.0-only`.

https://staff.vanillaforums.com/discussion/comment/1755/#Comment_1755